Library/src/Multiplayer/SpaceEntity.cpp

```diff
@@ -1191,6 +1247,52 @@ ComponentBase* SpaceEntity::FindFirstComponentOfType(ComponentType Type, bool Se
 	return LocatedComponent;
 }
 
+void SpaceEntity::AddChildEntitiy(SpaceEntity* ChildEntity)
+{
+	ChildEntities.Append(ChildEntity);
+}
+
+bool SpaceEntity::ResolveEntityHierarchy()
```

Comment:
Possibly better named `ResolveParent`? It's all about parentage, and it'd make the call-site on line #972 make more sense too.

Reply:
You're right, it updates lists of children on the parent too. To me that still feels appropriate to describe as parent resolution.

Reply:
Fair enough. Would you prefer to leave as-is?

Reply:
ResolveParentChildRelationship?

Reply:
I'm sold.

Reply:
Latest changes look good, thanks!